gossip/src/crds.rs

```diff
@@ -309,7 +323,10 @@ impl Crds {
                     Err(CrdsError::InsertFailed)
                 }
             }
-        }
+        };
+        //notify geyser interface
+        self.notify_clusterinfo_update(self.table.get(&gossip_label));
+        ret
```

Comment:
Crds::insert requires an exclusive lock on the crds table.
So we shouldn't put the notification code here.

Instead somewhere else, e.g. here:
https://github.com/anza-xyz/agave/blob/7629a152d/gossip/src/cluster_info.rs#L1618
we need to *read* locks crds table and obtain new entries since last time using a `crds::Cursor`.
similar to the code here:
https://github.com/anza-xyz/agave/blob/7629a152d/gossip/src/crds_gossip_push.rs#L184-L190

Reply:
alternatively somewhere in geyser can `pull` new contact-infos every once in a while or when needed.